Base/res/devel/templates/cpp-basic.ini

```diff
@@ -0,0 +1,4 @@
+[HSTemplate]
+Name=Command-line Application (C++)
+Description=Template for creating a basic C++ command-line application.
+Priority=95
```

Comment:
Missing newline at end of file - please configure your editor to add one. :)
(affects many files in this PR)

Reply:
I plan on running everything through clang-format before I squash and mark the PR as ready 😄